minicluster/src/main/java/tachyon/master/LocalTachyonClusterMultiMaster.java

```diff
@@ -98,7 +101,11 @@ public synchronized TachyonFS getClient() throws IOException {
   }
 
   public String getUri() {
-    return Constants.HEADER_FT + mCuratorServer.getConnectString();
+    return Constants.HEADER_FT + mHostname + ":" + mMasters.get(0).getMetaPort();
```

Comment:
why not replace `mMasters.get(0).getMetaPort()`  with `getMasterPort()`? the latter way is more intuitive to me.